writing.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<link rel="stylesheet" href="styles/main.css">
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title></title>
+  </head>
+  <header>
+    <h1>AC GILLETTE</h1>
+    <nav>
+      <a href="index.html">Home</a>
+      <a href="about.html">About</a>
+      <a href="projects.html">Projects</a>
+      <a href="writing.html">Writing</a>
+    </nav>
+  </header>
+  <body>
+    <article class="article">
```

Comment:
I like that you used a terminator face for each of your entries. I'm curious what or how you would like the image to appear once you start adding more text to the page. Depending on what you hope to accomplish there might be different ways to make this easier to implement.